src/pages/sites/sites.jsx

```diff
@@ -62,7 +80,31 @@ const SiteView = () => {
         })
         .catch(() => {});
     }
+
+    // FIX THIS TOO
```

Comment:
This function and the one below don't need to be included - we have added a new route which fetches all area data, the count of students, the count of sites, and the count of teachers